soapui/src/main/java/com/eviware/soapui/impl/rest/panels/request/views/content/GraphQLRequestContentView.java

```diff
@@ -134,7 +134,7 @@ public void release() {
     }
 
     @Override
-    public boolean supportsContentType(String contentType) {
-        return contentType.toLowerCase().endsWith("json");
+    public int getSupportScoreForContentType(String contentType) {
+        return contentType.toLowerCase().startsWith("application/json")? 2 : 0;
```

Comment:
I think the previous comparison is more correct. It is worked for the media types like text/json, application/vnd.api+json and so on.

Reply:
agreed

Reply:
@Alexander-Paeshin It didn't work, as content type here may contain a charset. I tried to use some classes from 3rd-party libs, like javax.mail and google, to parse it, but they threw exceptions. And I was afraid to add jakarta lib we use in ReadyAPI, as the transition from javax.mail to jakarta caused many issues in ReadyAPI. Do you think it's safer to look for ";" and throw off the charset manually?

Reply:
I think, yes. This is conform media-type specification.
https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.5
https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.1

Reply:
Changed to a common method.